lib/PuppeteerSharp/Browser.cs

```diff
@@ -22,34 +22,61 @@ public class Browser : IBrowser
         private readonly ConcurrentDictionary<string, BrowserContext> _contexts;
         private readonly ILogger<Browser> _logger;
         private readonly Func<TargetInfo, bool> _targetFilterCallback;
+        private readonly Func<TargetInfo, bool> _isPageTargetFunc;
         private readonly BrowserContext _defaultContext;
         private readonly CustomQueriesManager _customQueriesManager = new();
-
+        private readonly Func<Task> _closeCallback;
```

Comment:
This callback is also not invoked anywhere.

Reply:
Ticket created here https://github.com/hardkoded/puppeteer-sharp/issues/2100 :) Thanks!